README.md

```diff
@@ -165,7 +165,9 @@ is `User`.
 1. userSignUp
 1. userUpdatePassword
 1. userSendPasswordReset
+1. `userResendConfirmation(email: String!, redirectUrl: String!): UserResendConfirmationPayload`
```

Comment:
Can we keep the same structure as the other operations please? gql describe types for params and returns so no need to do it here. Also applies to line 170.

Reply:
Not sure what you are asking here. Review the current [mutation documentation](https://github.com/graphql-devise/graphql_devise#mutations). Am I doing something differently here?

Reply:
I see what you mean, I just got confused because the branch has not been rebased. Please disregard this comment.